src/jit/lsra.cpp

```diff
@@ -355,6 +355,35 @@ RegRecord* LinearScan::getRegisterRecord(regNumber regNum)
 }
 
 #ifdef DEBUG
+
+//----------------------------------------------------------------------------
+// getConstrainedRegMask: Returns new regMask which is the intersection of
+// regMaskActual and regMaskConstraint if the new regMask has at least 
+// minRegCount registers, otherwise returns regMaskActual.
```

Comment:
I remember discussing this with you.  Though considered, I haven't done partly based on your comment  that a stress mode doesn't have to be really "stressful".    Plus

1) even if a ref position of an interval is not stressed at a particular point,  it will be stressed at its next ref position where it is possible and that way we still will have coverage.
2) It will be a little more work and could slow down stress mode execution further.  I know we don't need to be concerned about increased execution time since it is a stress mode.
3) Not sure whether on x86 we have to be careful of what registers we remove from the set.  This needs to be experimented to figure out.